bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -1071,7 +1155,6 @@ private void onSelectionChanged(final ISelection selection) {
          // a tour was selected, get the chart and update the marker viewer
 
          final SelectionTourData tourDataSelection = (SelectionTourData) selection;
-         _tourData = tourDataSelection.getTourData();
```

Comment:
Why is `_tourData` removed, this changes the app logic

Reply:
I review always all my modifications before I commit my changes, also to find temporarily code.

Reply:
I typically do as well but evidently i forgot this time